### ISSUE TITLE
Linux desktop app loses appearance settings (theme)

### ISSUE THREAD

**Issue by ngrigoriev** [tutao/tutanota#4201]
- [X] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [X] I've searched and did not find a similar issue.

**Describe the bug**
Very often the desktop app resets to Light theme after restart after previously being forced into Dark color theme.

**To Reproduce**
1. Go to Settings -> Appearance
2. Click on 'Switch color theme'
3. Select 'Dark'
4. Use the application for a while
5. Restart the app or reboot and start the app. The app may (not always) start with the Light theme.

**Expected behavior**
Settings are properly persisted and, if applicable, migrated between the app upgrades 

**Screenshots**
Not needed

**Desktop (please complete the following information):**
 - Arch Linux / latest
 - i3 window manager, no desktop environment
 - 3.96.6

**Additional context**

**Comment by charlag** [comment]
If there some kind of a special permission on `.config` directory? Did you see anything suspicious in the output/logs (settings -> about -> send logs)?

**Comment by charlag** [comment]
Hm, the config is corrupted, that's not great. Did you try to delete it manually?

**Comment by ngrigoriev** [comment]
I am waiting for this to happen. However, the last update yesterday just killed tutanota :) It downloaded the update and then failed to install it. This was the last drop for me :) I have removed all left-overs, installed tutanota-desktop-bin  AUR package instead and disabled the automatic updates in the app itself. I am sure that this issue was connected to the automatic updates.

**Comment by charlag** [comment]
All of it sounds like a permission problem to be honest. I will close it for now, feel free to comment if you find out more.

**Comment by ngrigoriev** [comment]
It is certainly the auto-update that breaks it. It's been two weeks after I disabled the auto-update and the problem did not reoccur.

**Comment by ngrigoriev** [comment]
No, it is not (only) auto-update. I was just able to reproduce this even after having auto-update disabled for a while. But I think it is generally related to the updates. I have performed the normal update (arch linux) and then noticed that Tutanota was not running. I launched it and it started with the default settings. I closed it but then I noticed that it was still running in the background and did not want to die. I have also noticed that the newly launched app could not even write the logs. I had to kill the app processes running in the background (some with -9 because they did not want to quit peacefully ;) ). Then launched it again and noticed the following in the log:

2022-07-19T12:48:54.324Z W "sseInfo corrupted or not present, making sure pushEncSessionKeys and scheduled alarms are cleared"
...
2022-07-19T12:48:54.318Z E "[2022-07-19T12:48:54.315Z]","failed to read config!",?
2022-07-19T12:48:54.322Z I "config up to date"

By the way, the app loses ALL settings except the accounts, not just appearance.

I am certain that Tutanota simply does not handle the upgrade correctly and corrupts the configuration in some cases. This is certainly a very annoying bug.
